go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/onsi/gomega v1.27.6
 	github.com/open-policy-agent/cert-controller v0.7.0
 	github.com/prometheus/client_golang v1.15.1
+	github.com/ray-project/kuberay/ray-operator v0.0.0-20230516192117-9bc5d854bd31
```

Comment:
This is not a released version, please open an issue to track this, when kuberay published a new release, we should bump to it.